user_docs/en/userGuide.t2t

```diff
@@ -2784,18 +2784,18 @@ You can browse installed incompatible add-ons using the [incompatible add-ons ta
 + Extra Tools +[ExtraTools]
 
 ++ Log Viewer ++[LogViewer]
-The log viewer, found under Tools in the NVDA menu, allows you to view the logging output that has occurred since the last session of NVDA was started.
+The log viewer, found under Tools in the NVDA menu, allows you to view the logging output that has occurred since the current session of NVDA was started.

```

Comment:
I think "latest" is most accurate, to cover the case of after NVDA's session ends.

```suggestion
The log viewer, found under Tools in the NVDA menu, allows you to view the logging output that has occurred since the latest session of NVDA was started.
```